src/libs/LoginUtils.ts

```diff
@@ -101,4 +108,22 @@ function handleSAMLLoginError(errorMessage: string, cleanSignInData: boolean) {
     Navigation.goBack(ROUTES.HOME);
 }
 
-export {getPhoneNumberWithoutSpecialChars, appendCountryCode, isEmailPublicDomain, validateNumber, getPhoneLogin, areEmailsFromSamePrivateDomain, postSAMLLogin, handleSAMLLoginError};
+/**
+ * Check whether the current user has a login for a specific domain.
+ */
+function userHasLoginInDomain(domain: string): boolean {
```

Comment:
This method breaks the reactivity of the view component since the view component isn't connected to Onyx for the current user logins. Simple fix:

1. Remove this method
2. Use `useOnyx()` to connect to the user logins in the view component

Reply:
This is not needed anymore, we're going to let Auth throw a specific error if the user doesn't have access to the domain.